packages/flutter/test/painting/text_span_test.dart

```diff
@@ -245,21 +242,7 @@ void main() {
         null,
       ],
     );
-    FlutterError error;
-    try {
-      text.computeToPlainText(StringBuffer());
-    } on FlutterError catch (e) {
-      error = e;
-    }
-    expect(error, isNotNull);
-    expect(error.toStringDeep(),
-      'FlutterError\n'
-      '   TextSpan contains a null child.\n'
-      '   A TextSpan object with a non-null child list should not have any\n'
-      '   nulls in its child list.\n'
-      '   The full text in question was:\n'
-      '     TextSpan("foo bar")\n'
-    );
```

Comment:
looks like the point of this test was to check that our null checking gave a pretty error message. What's the error message now? Is it still pretty?

Reply:
It depends:

1. weak-mode, no null assertions - same error message
2. weak-mode, null assertions - null assertion error when constructing the TextSpan. Depending on the context that might be easy to read or ugly
3. strong-mode, static error can't be hit.